client/src/components/main/Nav/FusionTabs.tsx

```diff
@@ -0,0 +1,30 @@
+import { Box, Typography, makeStyles } from "@material-ui/core";
```

Comment:
The app does not compile due to single quotes rather than double quotes. You should resolve these errors in this file.

Reply:
rip I forgot a lot of this stuff. I commented out the rule locally. I am personally a fan of single quotes over double quotes, so I wanted to change that rule but wanted to discuss that with James when he gets back

Reply:
Changed to double quotes for now

Reply:
We prob just chose double because that's what we use in our python apps. I don't think it really matters here.